### ISSUE TITLE
nfs_export LWRP appends multiple anonids

### ISSUE THREAD

**Issue by lmickh** [atomic-penguin/cookbook-nfs#46]
I have code that looks something like this:

nfs_export "share1" do
  ...
  anonuser "user1"
  anongroup "group1"
end

nfs_export "share2" do
  ...
  anonuser "user2"
  anongroup "group2"
end

nfs_export "share3" do
  ...
  anonuser "user3"
  anongroup "group3"
end

The resulting /etc/exports file looks like this:

share1 *(rw,sync,root_squash,anonuid=user1,anongid=group1)
share2 *(rw,sync,root_squash,anonuid=user1,anongid=group1,anonuid=2,anongid=2)
share3 *(rw,sync,root_squash,anonuid=user1,anongid=group1,anonuid=2,anongid=2,anonuid=3,anongid=3)

**Comment by atomic-penguin** [comment]
Looks to be a bug in #15.  I believe the following change will fix this.

```
diff --git a/providers/export.rb b/providers/export.rb
index dcb065cd9891..ff3f7e0f888b 100644
--- a/providers/export.rb
+++ b/providers/export.rb
@@ -27,14 +27,10 @@ action :create do
 
     ro_rw = new_resource.writeable ? 'rw' : 'ro'
     sync_async = new_resource.sync ? 'sync' : 'async'
-    if new_resource.anonuser
-      new_resource.options << "anonuid=#{find_uid(new_resource.anonuser)}"
-    end
-    if new_resource.anongroup
-      new_resource.options << "anongid=#{find_gid(new_resource.anongroup)}"
-    end
     options = new_resource.options.join(',')
     options = ",#{options}" unless options.empty?
+    options << ",anonuid=#{find_uid(new_resource.anonuser)}" if new_resource.anonuser
+    options << ",anongid=#{find_gid(new_resource.anongroup)}" if new_resource.anongroup
 
     if new_resource.network.is_a?(Array)
       host_permissions = new_resource.network.map { |net| net + "(#{ro_rw},#{sync_async}#{options})" }
```

**Comment by atomic-penguin** [comment]
Thanks for reporting.  I have pushed fb32702 to address this.  I might not get a release out to the community site tonight, as I have pending issues to review before release.

**Comment by atomic-penguin** [comment]
Released and fixed in 2.1.0, forgot to put it in the CHANGELOG for release.
